tests/test_workers/test_tasks/test_opm_operations.py

```diff
@@ -166,9 +167,13 @@ def test_opm_migrate(
 
 @pytest.mark.parametrize("dockerfile", (None, 'index.Dockerfile'))
 @mock.patch('iib.workers.tasks.utils.run_cmd')
-def test_opm_generate_dockerfile(mock_run_cmd, tmpdir, dockerfile):
+@mock.patch('iib.workers.tasks.opm_operations.generate_cache_locally')
+@mock.patch('iib.workers.tasks.opm_operations.insert_cache_into_dockerfile')
+@mock.patch('iib.workers.tasks.opm_operations.verify_cache_insertion_into_dockerfile')
+def test_opm_generate_dockerfile(mock_vciid, mock_icid, mock_gcl, mock_run_cmd, tmpdir, dockerfile):
```

Comment:
I am not sure if we should test also `mock_vciid` 
The test is missing so we did not discover the incorrect reading of Dockerfile.